.changeset/slow-zebras-clap.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/backstage-plugin-github-pull-requests': major
```

Comment:
```suggestion
'@roadiehq/backstage-plugin-github-pull-requests': minor
```
Minor version bump should be enough as we do not break the API.

Reply:
Done 👍